userspace/libsinsp/sinsp.h

```diff
@@ -219,14 +219,14 @@ class SINSP_PUBLIC sinsp : public capture_stats_source, public wmi_handle_source
 
 
 	/* Wrappers to open a specific engine. */
-	void open_kmod(uint64_t buffer_dimension, const std::unordered_set<uint32_t> &ppm_sc_of_interest = {}, const std::unordered_set<uint32_t> &tp_of_interest = {});
-	void open_bpf(uint64_t buffer_dimension, const char* bpf_path, const std::unordered_set<uint32_t> &ppm_sc_of_interest = {}, const std::unordered_set<uint32_t> &tp_of_interest = {});
+	void open_kmod(unsigned long driver_buffer_bytes_dim = DEFAULT_DRIVER_BUFFER_BYTES_DIM, const std::unordered_set<uint32_t> &ppm_sc_of_interest = {}, const std::unordered_set<uint32_t> &tp_of_interest = {});
```

Comment:
Perhaps passing `0` as default value for `driver_buffer_bytes_dim`  would help us to avoid declaring `DEFAULT_DRIVER_BUFFER_BYTES_DIM` here and in sinsp example (defautl value would be just 0) and in scap-open example (same)?
Moreover, it would allow us to define a `DRIVER_BUFFER_BYTES_DIM` macro in just one place (likely `ppm_ringbuffer.h`) and include it in scap and in kmod.